### PR TITLE
make use of the correct qt in extra buoyancy

### PR DESCRIPTION
**Pull request by yairchn**
Behavioral changes but plots look fine

### CODE REVIEW

**Comment by charleskawczynski** [comment]
> Behavioral changes but plots look fine

They don't look fine to me! Bomex updraft w looks like it's zero!

File: src/EDMF_Updrafts.jl [hunk 1]
```diff
@@ -393,11 +393,10 @@ function buoyancy(
...
                     UpdVar.RH.values[i, k] = relative_humidity_c(self.Ref.p0_half[k], qt, qt - qv, 0.0, t)
                 elseif UpdVar.Area.values[i, k - 1] > 0.0 && k > kc_surf
-                    qt = UpdVar.QT.values[i,k-1]
-                    h  = UpdVar.H.values[i,k-1]
+                    qt = UpdVar.QT.values[i, k - 1]
+                    h = UpdVar.H.values[i, k - 1]
                     sa = eos(self.Ref.p0_half[k], qt, h)
-                    qt -= sa.ql
-                    qv = qt
+                    qv = qt = sa.ql
```
Comment (charleskawczynski) [inline src/EDMF_Updrafts.jl:399]:
```suggestion
                    qv = qt - sa.ql
```

**Review by trontrytel — APPROVED** [review]
The only difference I see is in the dry bubble case. But I don't know if it was always different between scampy and TC.jl or if it's caused by this change?

**Review by charleskawczynski — APPROVED** [review]
Whatever I saw before seems to be fixed 🎉

**Comment by trontrytel** [comment]
> Whatever I saw before seems to be fixed 🎉

What has been seen, cannot be unseen...

**Comment by yairchn** [comment]
Bors r+

**Pull request merged**